app/dashboard/views.py

```diff
@@ -3659,9 +3664,10 @@ def hackathon(request, hackathon='', panel='prizes'):
         'orgs': orgs,
         'keywords': json.dumps([str(key) for key in Keyword.objects.all().values_list('keyword', flat=True)]),
         'hackathon': hackathon_event,
+        'hacker_count': hacker_count,
+        'projects_count': projects_count,
         'hackathon_obj': HackathonEventSerializer(hackathon_event).data,
-        'is_registered': json.dumps(True if is_registered else False),
-        'hackathon_not_started': hackathon_not_started,
+        'is_registered': True if is_registered else False,
```

Comment:
this will not get into document.is_registered === 'True' (string instead of boolean?)

Reply:
Nah it gets converted to json Boolean. 

This is the way django suggests now